packages/rich-text/src/component/index.js

```diff
@@ -58,6 +57,35 @@ const INSERTION_INPUT_TYPES_TO_IGNORE = new Set( [
 	'insertLink',
 ] );
 
+/**
+ * In HTML, leading and trailing spaces are not visible, and multiple spaces
+ * elsewhere are visually reduced to one space. This rule prevents spaces from
+ * collapsing so all space is visible in the editor and can be removed. It also
+ * prevents some browsers from inserting non-breaking spaces at the end of a
+ * line to prevent the space from visually disappearing. Sometimes these non
+ * breaking spaces can linger in the editor causing unwanted non breaking spaces
+ * in between words. If also prevent Firefox from inserting a trailing `br` node
+ * to visualise any trailing space, causing the element to be saved.
+ *
+ * > Authors are encouraged to set the 'white-space' property on editing hosts
+ * > and on markup that was originally created through these editing mechanisms
+ * > to the value 'pre-wrap'. Default HTML whitespace handling is not well
+ * > suited to WYSIWYG editing, and line wrapping will not work correctly in
+ * > some corner cases if 'white-space' is left at its default value.
+ *
+ * https://html.spec.whatwg.org/multipage/interaction.html#best-practices-for-in-page-editors
+ *
+ * @type {string}
+ */
+const whiteSpace = 'pre-wrap';
+
+/**
+ * Default style object for the editable element.
+ *
+ * @type {Object}
```

Comment:
For future consideration: As a type, `Object` is better than nothing, but if we want to move toward better type enforcement (e.g. #17014), we probably want to be more specific about the sorts of values we're expecting to exist in the object.

```suggestion
 * @type {Object<string,string>}
```

In this case, it doesn't matter as much since it's a constant value ([aside: naming](https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#constants)), but can be helpful for when possibly assigning into or consuming from an object to have the type awareness / enforcement.

Reply:
I don't know that much about it. Why is `string` listed twice in `<string,string>`?

Reply:
I find the following can be good for reference:

https://jsdoc.app/tags-type.html
https://www.typescriptlang.org/docs/handbook/type-checking-javascript-files.html#supported-jsdoc

There are two mentions of `string` to respectively represent the type of the key and the type of the value. (Which on reflection is a bit odd, since objects can only have string keys anyways)

Reply:
Yeah, that is odd! :)

Reply:
Coming back to this, I discovered why it is useful to be able to define the type of the key. While true that an object key is always a string, we can do things like create enums of possible string options using a custom type.

See example at: https://github.com/WordPress/gutenberg/pull/18816#discussion_r353808652

Reply:
Nice!